Gemfile.lock

```diff
@@ -0,0 +1,122 @@
+PATH
```

Comment:
@albertjou Fotini said she's for putting it back in because rubocop has been upgraded to a newer version and failed the CI build.